crates/core/tedge_api/src/entity_store.rs

```diff
@@ -939,114 +940,64 @@ pub enum InitError {
     Custom(String),
 }
 
-/// An object representing a valid entity registration message.
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-pub struct EntityRegistrationMessage {
-    // fields used by thin-edge locally
+pub struct EntityRegistrationPayload {
```

Comment:
A doc comment would help to understand why we have two similar structs, aka `EntityRegistrationMessage` and `EntityRegistrationPayload` and what are their relationship.

Reply:
Actually I failed to understand why we need `EntityRegistrationPayload`. If it needs for serde, we can implement custom Serialize/Deseralize for `EntityRegistrationMessage`...

Reply:
Resolved by [8654090](https://github.com/thin-edge/thin-edge.io/pull/3587/commits/8654090f76cf664e56dff2f493927e9a1b324c18)